cpp/benchmarks/io/json/json_reader_input.cpp

```diff
@@ -31,10 +31,12 @@ constexpr cudf::size_type num_cols = 64;
 
 void json_read_common(cuio_source_sink_pair& source_sink,
                       cudf::size_type num_rows_to_read,
-                      nvbench::state& state)
+                      nvbench::state& state,
+                      cudf::io::compression_type comptype = cudf::io::compression_type::NONE,
+                      size_t datasize                     = data_size)
```

Comment:
Super nit: We could've gotten away with:
```suggestion
                      size_t data_size                     = data_size)
```
It's not like the outer `data_size` is used anywhere in the function. :]

Reply:
Minor non-binding nit: I say we should just rename the global one to say `data_size_default` or something to differentiate.